__tests__/app.test.js

```diff
@@ -65,6 +65,7 @@ describe("2 GET reviews by id /api/reviews/:review_id", () => {
         expect(body).toEqual({
           review: {
             category: "euro game",
+            comment_count: 0,
```

Comment:
better to have a new test for updating rather than modify the old test.
tips: using expect.objectContaining for this ticket's test can ignore the extra keys